tests/Utils/BuildClientSchemaTest.php

```diff
@@ -0,0 +1,1082 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GraphQL\Tests\Utils;
+
+use GraphQL\GraphQL;
+use GraphQL\Type\Definition\EnumType;
+use GraphQL\Type\Definition\ObjectType;
+use GraphQL\Type\Definition\Type;
+use GraphQL\Type\Introspection;
+use GraphQL\Type\Schema;
+use GraphQL\Utils\BuildClientSchema;
+use GraphQL\Utils\BuildSchema;
+use GraphQL\Utils\SchemaPrinter;
+use PHPUnit\Framework\TestCase;
+use function array_filter;
+
+/**
+ * @see BuildClientSchema
+ */
+class BuildClientSchemaTest extends TestCase
```

Comment:
final, should not be allowed to be extended

Reply:
I am firmly opposed to the usage of the `final` keyword in libraries such as this.

Reply:
I understand there are people avoiding `final` but for tests there's no issue, hm? Consumers won't get affected anyway and we accidentally won't extend Tests nor they will get suggested in autocomplete.